### DISCUSSION TITLE
嵌入式输入时，word中打字输入文字的地方会闪烁

### DISCUSSION THREAD

**Discussion by zllx666** [rime/home#954] [Q&A]
想请教一下有人遇到这个问题吗？我是win10 21H2，最新版rime，excel和powerpoint都是好的
和这个issue是一样的问题：[https://github.com/rime/weasel/issues/538](url)

**Comment by MeoteCarlo** [comment]
我也遇到了，还没找到解决办法，虽然word用得不多，但也比较困扰。感觉word的问题多一些？

**Comment by zllx666** [comment]
不知道为啥  我就不用嵌入式输入了

**Comment by rtransformation** [comment]
用word倒是没遇到过这问题，我用OneNote的时候，Rime倒总是显示不出来选词框。

**Comment by xtellaris** [comment]
在 Windows11 23H2 实测到一个奇怪的情况：

1. vscode+[vim插件](https://marketplace.visualstudio.com/items?itemName=vscodevim.vim)+Rime开启嵌入式输入，输入时会候选窗格会闪烁
2. vscode+关闭vim插件+Rime开启嵌入式输入，候选窗格不闪烁

猜测这个问题也许和Windows窗口的一些检测输入法的行为有关，不知Rime做适配是否有难度
